cdap-operational-stats-core/pom.xml

```diff
@@ -120,6 +124,14 @@
       <artifactId>jcl-over-slf4j</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
```

Comment:
Is there any specific functionalities you needed from the httpclient that is missing from the java `HttpURLConnection`? If not, please use the Java one instead. There is also a set of classes in CDAP for simplifying the interaction. Find usage of the `HttpRequests` class as a starting point.